### DISCUSSION TITLE
Simple Bouncy Girl Animation

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#135] [Show and tell]
As I was playing with ToonSquid I decided to make the Salamanca brush from Procreate. I liked how it looked so I made quick animations playing with it. Here is one of them.

https://user-images.githubusercontent.com/88890833/213809475-094958e3-43c7-482c-a918-9cace43f4b47.mov

Maybe post your animations too.

**Comment by trinhpxl** [comment]
this is so cute !! i'll post an animation i made recently :D
